docs/guide/snaps-introduction.md

```diff
@@ -0,0 +1,69 @@
+# Snaps Tutorial
+
+## What is Snaps?
+
+Snaps are a new evolution of MetaMask's functionality. By using MetaMask Flask in conjunction with the Snaps API, developers are able to bring their own code to the wallet in a custom-made code laboratory. MetaMask is the first wallet to provide this openness and flexibility to developers, paving the way to scale the number of features and personalization available to the wallet and your Dapp.
```

Comment:
In the first three sentences, we've described snaps as "a new evolution", and as a "custom-made code laboratory", and congratulated ourselves on being the first to create such a thing, all without saying what it is. This does not seem helpful. We should be explaining what snaps is, not trying to sell it and the wallet itself.

Reply:
So, what do you propose? Do we cut this copy entirely?

Reply:
This is what I have so far for this section:

>Snaps is a system that allows safely expanding the capabilities MetaMask. A "snap" is a program that we run in an isolated environment that can customize the wallet experience in some way. For example, a snap can add new APIs to MetaMask, add new features, or modify existing functionality using internal APIs. Snaps makes extending MetaMask easier and safer, for both users and developers.

Reply:
I'm OK with this if you guys are.